java/src/jmri/jmrit/operations/JmritOperationsBundle_da.properties

```diff
@@ -81,8 +86,9 @@ South               = Syd
 AM                  = AM
 PM                  = PM
 
-
 # change the order in tables
 Up                        = Op
 Down                      = Ned
-Status=Status
+
+# no need to translate the following line. Will be picked up by any locale from here:
+ReservedChar        = . < > : " / \\ | ? *
```

Comment:
Like it says in the comment, you don't need to copy this key/line to your language. It even adds a risk for differences. Advice to leave this out.